docs/introduction_tutorial/intro_tutorial_part1.ipynb

```diff
@@ -102,9 +102,9 @@
     "### Create a URL Template String\n",
```

Comment:
<p>I like the way you made URL_FORMAT a variable!  I think it makes what is happening in the return value clearer.</p><p>The one change I would make would be to define the URL_FORMAT variable inside the function. Otherwise a user could change URL_FORMAT outside the function and change what happens inside the function.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-forge/pangeo-forge-recipes/pull/343/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>True, the user could change the value. I only made it global because there is a convention in python I think to put constant variables in all-caps in the global namespace. It makes it more explicit what things are constants and what are not in code, and avoids reallocating the string every time it is used. Also, other modules can then import this URL_FORMAT if needed somewhere else. But I'm happy to move the string inside the function if you prefer :)</p>

Reply:
Okay, I see your logic there. 👍🏻